### PR TITLE
Add hotkeys on product page

### PR DESCRIPTION
**Pull request by antho-girard**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | This PR adds hotkeys on product page: ⌘+S, ⌘+D, ⌘+P, ⌘+V, ⌘+O |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1329 |
| How to test? | On product page, try the different shortcuts above |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -925,18 +925,30 @@ var form = (function() {
         e.preventDefault();
         send();
       });
+      jwerty.key('cmd+S', function(e) {
```
Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/product/form.js]: The idea is good, but we have duplicated code ... Can you please merge the Mac hotkeys in the existing call ?

```js
jwerty.key('ctrl+S/cmd+S', function(e) {
```

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 2]
```diff
@@ -925,18 +925,30 @@ var form = (function() {
...
+        e.preventDefault();
+        send();
+      });
 
       /** create keyboard event for save & duplicate */
       jwerty.key('ctrl+D', function(e) {
         e.preventDefault();
         send($('.product-footer .duplicate').attr('data-redirect'));
       });
+      jwerty.key('cmd+D', function(e) {
```
Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/product/form.js]:
```js
jwerty.key('ctrl+D/cmd+D', function(e) {
```

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 3]
```diff
@@ -925,18 +925,30 @@ var form = (function() {
...
+        e.preventDefault();
+        send($('.product-footer .duplicate').attr('data-redirect'));
+      });
 
       /** create keyboard event for save & new */
       jwerty.key('ctrl+P', function(e) {
         e.preventDefault();
         send($('.product-footer .new-product').attr('data-redirect'));
       });
+      jwerty.key('cmd+P', function(e) {
```
Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/product/form.js]:
```js
jwerty.key('ctrl+P/cmd+P', function(e) {
```

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 4]
```diff
@@ -950,13 +962,23 @@ var form = (function() {
...
+        send(productFooter.attr('data-redirect'), productFooter.attr('target'));
+      });
 
       /** create keyboard event for save & active or desactive product*/
       jwerty.key('ctrl+O', function(e) {
         e.preventDefault();
         var step1CheckBox = $('#form_step1_active');
         step1CheckBox.prop('checked', !step1CheckBox.is(':checked'));
       });
+      jwerty.key('cmd+O', function(e) {
```
Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/product/form.js]:
```js
jwerty.key('ctrl+O/cmd+O', function(e) {
```

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 5]
```diff
@@ -950,13 +962,23 @@ var form = (function() {
           var productFooter = $('.product-footer .preview');
           send(productFooter.attr('data-redirect'), productFooter.attr('target'));
       });
+      jwerty.key('cmd+V', function(e) {
```
Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/product/form.js]:
```js
jwerty.key('ctrl+V/cmd+V', function(e) {
```

Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/product/form.js]:
```js
jwerty.key('ctrl+V/cmd+V', function(e) {
```

**Review by Quetzacoalt91 — DISMISSED** [review]
Yep, we should add this feature for Mac OS users.

However, in order to avoid duplicated code, it would be great to merge the different hotkeys possibilities in the same call to the `key()`function.

**Comment by mickaelandrieu** [comment]
ping @antho-girard, would you mind to add "Windows/World?" support to your hotkeys ? :)

**Comment by antho-girard** [comment]
Hi,

Yes, I'll do it tonight.
Sorry for the delay :(

**Comment by antho-girard** [comment]
Hi @mickaelandrieu 

It should be ok now.

Thanks !

**Comment by mickaelandrieu** [comment]
Thank you @antho-girard !

**Pull request merged**